scripts/container-postCreate.sh

```diff
@@ -0,0 +1,6 @@
+#/bin/bash
+
+# Run pip install
+pip install -r requirements.txt
```

Comment:
Just curious, why do we use a post-create script in stead of installing in the dockerfile?

I'm thinking WRT to caching, does this command get cached? AFAIK docker caches if we don't change anything above this command

Reply:
The docker container is not built with the code. The code is mounted in during container init, hence nothing to run pip install on.

As to why we dont include the requirement.txt into the docker, thats not the ideal practice with devcontainers. Different discussion if we are building for release.

Reply:
Need to read up on how this works, any best references you recommend?